app/Main.hs

```diff
@@ -1,6 +1,11 @@
 module Main where
 
-import Lib
+import           HaskellWorks.Data.Network.Ip as I
 
 main :: IO ()
-main = someFunc
+main = do
+  let srcFile = "/Users/jwan/whole_internet.txt.blks"
```

Comment:
I guess this will only work on your computer :)

Reply:
I'll remove this, thanks :)